appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -38,7 +38,7 @@ title Attestations
 # Show a shortened version of the attestations to avoid excessive logging
 echo -n "$attestations" | jq -r '.[].predicate = "..."'
```

Comment:
Might need to tweak the jq query here, e.g.:

    jq -r '.attestations.[].predicate = "..."'

Reply:
Correction from your new revision: `jq -r '.attestations[].predicate = "..."'`